bindings/c/test/apitester/TesterCorrectnessWorkload.cpp

```diff
@@ -125,6 +134,66 @@ class ApiCorrectnessWorkload : public ApiWorkload {
 		    });
 	}
 
+	void randomGetKeyOp(TTaskFct cont) {
+		int numKeys = Random::get().randomInt(1, maxKeysPerTransaction);
+		auto keysWithSelectors = std::make_shared<std::vector<std::pair<fdb::Key, fdb::KeySelector>>>();
+		auto selectors = std::make_shared<std::vector<fdb::KeySelector>>();
+		auto results = std::make_shared<std::vector<fdb::Key>>();
+		for (int i = 0; i < numKeys; i++) {
+			auto key = randomKey(readExistingKeysRatio);
+			fdb::KeySelector selector;
+			selector.orEqual = Random::get().randomBool(0.5);
+			selector.offset = Random::get().randomInt(0, 4);
+			keysWithSelectors->push_back({ key, selector });
+		}
+		execTransaction(
+		    [keysWithSelectors, results](auto ctx) {
+			    auto futures = std::make_shared<std::vector<fdb::Future>>();
+			    for (const auto& keyWithSelector : *keysWithSelectors) {
+				    auto key = keyWithSelector.first;
+				    auto selector = keyWithSelector.second;
+				    selector.key = key.data();
+				    selector.keyLength = key.size();
+				    futures->push_back(ctx->tx().getKey(selector, false));
+			    }
+			    ctx->continueAfterAll(*futures, [ctx, futures, results]() {
+				    results->clear();
+				    for (auto& f : *futures) {
+					    results->push_back(fdb::Key(f.get<fdb::future_var::KeyRef>()));
+				    }
+				    ASSERT(results->size() == futures->size());
+				    ctx->done();
+			    });
+		    },
+		    [this, keysWithSelectors, results, cont]() {
+			    ASSERT(results->size() == keysWithSelectors->size());
+			    for (int i = 0; i < keysWithSelectors->size(); i++) {
+				    auto const& key = (*keysWithSelectors)[i].first;
+				    auto const& selector = (*keysWithSelectors)[i].second;
+				    auto expected = store.getKey(key, selector.orEqual, selector.offset);
+				    auto actual = (*results)[i];
+				    // Local store only contains data for the current client, while fdb contains data from multiple
+				    // clients. If getKey returned a key outside of the range for the current client, adjust the result
+				    // to match what would be expected in the local store.
+				    if (actual.substr(0, keyPrefix.size()) < keyPrefix) {
+					    actual = store.startKey();
+				    } else if ((*results)[i].substr(0, keyPrefix.size()) > keyPrefix) {
+					    actual = store.endKey();
+				    }
+				    if (actual != expected) {
+					    error(fmt::format("randomGetKeyOp mismatch. key: {}, orEqual: {}, offset: {}, expected: {:.80} "
```

Comment:
Probably don't need to truncate with {.80} here, as these are keys and not values

Reply:
Good point, done.